test/unit/SparkPostTest.php

```diff
@@ -1,15 +1,16 @@
 <?php
 namespace SparkPost\Test;
 
-use SparkPost\SparkPost;
 use Ivory\HttpAdapter\CurlHttpAdapter;
+use Mockery;
+use SparkPost\SparkPost;
 use SparkPost\Test\TestUtils\ClassUtils;
-use \Mockery;
 
 class SparkPostTest extends \PHPUnit_Framework_TestCase {
 
   private static $utils;
   private $adapterMock;
+  /** @var SparkPost */
```

Comment:
It was just to specify that `$resource` has `SparkPost` object inside to remove some warnings in code (otherwise IDE doesn't know which type has `$this->resource`)